### ISSUE TITLE
Other metrics than accuracy

### ISSUE THREAD

**Issue by dafnevk** [NLeSC/mcfly#128]
Now we decide what the best model is based on accuracy. It should be possible for the user to define a different metric, e.g. categorical cross-entropy loss.

**Comment by dafnevk** [comment]
This happens in the model construction in `modelgen.py`. See: https://keras.io/metrics/

**Comment by williamwongys** [comment]
I use a euclidean distance metric to select the best model:
dist = math.sqrt((acc-1.0)**2 + (val_acc-1.0)**2)

So ideally the model should perform best for both training and validation data (min dist)
Does it make sense?

**Comment by cwmeijer** [comment]
Pull request with fix was already merged. Closing.
